src/backend/distributed/planner/multi_explain.c

```diff
@@ -150,6 +154,17 @@ CoordinatorInsertSelectExplainScan(CustomScanState *node, List *ancestors,
 							   "... SELECT commands via the coordinator")));
 	}
 
+	PlannedStmt *selectPlan = pg_plan_query(query, cursorOptions, params);
+	if (IsRedistributablePlan(selectPlan->planTree) &&
+		IsSupportedRedistributionTarget(targetRelationId))
+	{
+		ExplainPropertyText("INSERT/SELECT method", "repartition", es);
+	}
+	else
+	{
+		ExplainPropertyText("INSERT/SELECT method", "route via coordinator", es);
```

Comment:
Maybe "pull to coordinator". 

"route" is probably more correct, but somehow seems a bit less clear, mostly because it makes me think of router queries.

Reply:
done